recipes/xclip/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "xclip" %}
+{% set version = "0.13" %}
+{% set sha256 = "ca5b8804e3c910a66423a882d79bf3c9450b875ac8528791fb60ec9de667f758" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/astrand/xclip/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  detect_binary_files_with_prefix: true
+  run_exports:

```

Comment:
This package only contains an executable, not a shared library, so it shouldn't have a `run_exports`.